The Great Deck 3 Remap

SleepyGemmy opened the pull request:
this PR remaps the central ring on deck 3 and the recreational area to make it seem more like a ship that people live in under a longer period of time.

SleepyGemmy commented:
!review

JohnWildkins reviewed (approved):
have you heard of our lord and saviour atomic pr

SleepyGemmy commented:
![waa](https://user-images.githubusercontent.com/99297919/167097273-c818a4f6-ac79-4aae-9d9c-4c810c038a33.png)

TheDanseMacabre reviewed (changes requested):
Really, really not a fan of this remap. It removes excellent areas of open space that remind us of the verticality of the map.
The good/okay:
I suppose I don't mind the new cafe area (except for the fact it blocks the tree from above), bunker remap, and the command changes. I thought the little "nature showcase" room was cute. The new crew armory is good.
The bad:
I do not understand why more than half the open space on the map has been removed. In fact, I hate that most of it has been removed. I think the open space is an excellent design choice - it adds greater verticality and freedom of movement (as much as I loathe people treating the ship like a jungle gym on code green), and all in all it *looks good*. One could argue the open space serves no purpose - I would argue that this remap mostly replaces it with things that serve the same amount of purpose but do so in a way that doesn't look nearly as good. Here's the fact of the matter: these changes make the ship much more claustrophobic. I find myself looking at it and being reminded of the Aurora, not the Horizon.
Moving on from that - there's a lot of additions that serve no purpose except to be pointless fluff - I don't mind pointless fluff by any means - it is a good thing, at times, but in this case I think some of it detracts from the map. There's four tiles of water and a few grass tiles where the open space near the vacant office used to be - in operations, half the open space was removed to make way for a... table and two chairs. Cryogenics was, against all logic, made huge. There's a bunch of these tiny little spaces that for people to roleplay in - when we already have so many of those that most are seldom used.

I don't think this should be merged without a feedback thread beforehand. I think this PR makes too many changes.

anconfuzedrock commented:
two things I really don't like about new operations, having played a lot of hangar tech *on this map*:
-forget the elevator being smaller, I dislike that it uses airlocks. Airlocks can't be permanently opened, which is really handy for pushing crates off when between you and the door.
-I don't like the colored floor decal boxes. I think they're annoying and gaudy if you're doing anything except sorting the warehouse in that exact way, by department. and what if I don't want to sort the warehouse by department? plenty of other ways to sort it. and if you are, you are often delivering them. Even if I do though it's just a bit ugly.

Not much comment on the other parts. I much prefer the open hangar but other people have said that. those two things though I think will be annoying and I doubt I'll ever get used to full airlock 2x2 lift. it'll be real cramped. also maybe give it doors in the same spots on the up and on the down so you get trapped? there's four doors on the up, 2 on the down. no reason the bottom can't also have four doors. I think that at least would make it better.

Mail chute in the warehouse? I'm conflicted but I think it's cool. belt's longer than it needs to be though. It feels less realistic as a warehouse when it has less space and it feels like the entire area is designed to make shipping the stuff in it out as fast as possible instead of anything else. More efficient aint always better.

Afya212 commented:
AI ere: Core starts unpowered, also a lot of random important blindspots; https://i.imgur.com/MRBmoE4.png; https://i.imgur.com/MRBmoE4.png; https://i.imgur.com/gG0BufL.png; https://i.imgur.com/gG0BufL.png; https://i.imgur.com/QhjZV4v.png; https://i.imgur.com/QhjZV4v.png; https://i.imgur.com/hGHNx8j.png

Also quite a few areas on deck 2, but unsure how much of it is related to this PR.

Afya212 commented:
AI ere: Core starts unpowered, also a lot of random important blindspots: https://i.imgur.com/MRBmoE4.png; https://i.imgur.com/FePqghH.png; https://i.imgur.com/MKpJi6X.png; https://i.imgur.com/QhjZV4v.png; https://i.imgur.com/hGHNx8j.png; https://i.imgur.com/aYt2pCR.png; https://i.imgur.com/I2zrsz2.png

Also quite a few areas on deck 2, but unsure how much of it is related to this PR.

I gotta say, I love the new command bunker plus the cute hidden pathways in general.

Afya212 reviewed (changes requested):
AI ere: Core starts unpowered, also a lot of random important blindspots: https://i.imgur.com/MRBmoE4.png; https://i.imgur.com/FePqghH.png; https://i.imgur.com/MKpJi6X.png; https://i.imgur.com/QhjZV4v.png; https://i.imgur.com/hGHNx8j.png; https://i.imgur.com/aYt2pCR.png; https://i.imgur.com/I2zrsz2.png

Also quite a few areas on deck 2, but unsure how much of it is related to this PR.

I gotta say, I love the new command bunker plus the cute hidden pathways in general.

SleepyGemmy commented:
RE: [comment.](https://github.com/Aurorastation/Aurora.3/pull/13743#issuecomment-1120353968)
\----
> I dislike that it uses airlocks. Airlocks can't be permanently opened, which is really handy for pushing crates off when between you and the door.

addressed. the deck 1 operations elevator now has a second set of doors and all of its airlocks no longer close automatically.

> also maybe give it doors in the same spots on the up and on the down so you get trapped? there's four doors on the up, 2 on the down. no reason the bottom can't also have four doors. I think that at least would make it better.

addressed.

> Mail chute in the warehouse?

it's a chute _to_ the mail room, it's not a mail chute. it's an alternative to dragging crates up the stairs or the elevator.

> belt's longer than it needs to be though.
addressed. made the belt 1 tile shorter.

Llywelwyn commented:
this stuff should probably be made in an actual feedback thread on the forums somewhere, but just 2 give my 2c on the operations stuff:

>-I don't like the colored floor decal boxes. I think they're annoying and gaudy if you're doing anything except sorting the warehouse in that exact way, by department. and what if I don't want to sort the warehouse by department? plenty of other ways to sort it. and if you are, you are often delivering them. Even if I do though it's just a bit ugly.

in my experience, sorting by department is definitely the overwhelming majority of how people decide to sort the warehouse (i can't really think of any alternate ways i've seen it done). as it stood previously, it was frustrating to do the warehouse as a duo because you'd have to consult one other on which pile was which. after playing with this as a test-merge, i've been able to immediately jump into helping someone out and we've both immediately known, from the squares on the floor, what belongs where. in that sense, i think this change is conducive to rp and being able to work together in there. it _could_ probably be made to look better, but i guess that's why this is a test-merge.
 
> Mail chute in the warehouse? I'm conflicted but I think it's cool. belt's longer than it needs to be though. It feels less realistic as a warehouse when it has less space and it feels like the entire area is designed to make shipping the stuff in it out as fast as possible instead of anything else. More efficient aint always better.

i think with how the elevator and mailroom are right now it's a necessity. it takes objectively longer for mail to get where it needs to be when using that chute rather than the mailroom upstairs (only by like ~20 seconds or so), but saves the frustrating busywork of dragging each crate 1 at a time up a floor.

the amount of people who even bother to sort the warehouse dropped _really_ dramatically moving from aurora to the horizon, from what i've experienced, so evidently there is some sort of problem or frustration people are having with doing it. i couldn't say why other than my personal experience: it's incredibly annoying having to mess around with the elevator and hopping between floors for every single crate or bounty. i don't think making it easier is bad in this case.

SleepyGemmy commented:
RE: [comment.](https://github.com/Aurorastation/Aurora.3/pull/13743#pullrequestreview-965430325)
\----
> AI ere: Core starts unpowered, also a lot of random important blindspots: [...]

fixed!

Tristan63 commented:
This is a great redesign, however it would be nice if the mining shuttle pod bay was extended by 1 on all sides so you can do a walkaround of the shuttle, great incase repairs are required etc.

SleepyGemmy commented:
RE: [comment.](https://github.com/Aurorastation/Aurora.3/pull/13743#issuecomment-1120637583)
\----


> This is a great redesign, however it would be nice if the mining shuttle pod bay was extended by 1 on all sides so you can do a walkaround of the shuttle, great incase repairs are required etc.

done.
